src/Commands/ArtCommand.php

```diff
@@ -10,62 +10,61 @@
 
 class ArtCommand extends TerminusCommand
 {
+    protected $filename;
+
     /**
      * Displays Pantheon ASCII artwork
      *
      * @command art
      *
      * @param string $name Name of the artwork to select
+     *
      * @usage terminus art rocket
      *   Displays the rocket artwork
      */
-    public function art($name) {
-        $artwork_content = $this->retrieveArt($name);
-        $this->io()->text("<comment>{$artwork_content}</comment>");
+    public function art($name = '')
+    {
+        $this->setFilename($name);
+        if ( ! empty($name)) {
+            $artwork_content = $this->retrieveArt($name);
+            $this->io()->text("<comment>{$artwork_content}</comment>");
+        }
     }
 
     /**
      * If the user does not specify the $name parameter, then we will
      * prompt for it here.
      *
      * @hook interact
+     *
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
      */
     public function interact(InputInterface $input, OutputInterface $output)
     {
         $available_art = $this->availableArt();
-
-        $io = new SymfonyStyle($input, $output);
-        $art_name = $input->getArgument('name');
-        if (!$art_name) {
-            $art_name = $io->choice('Select art:', $available_art, 'fist');
-            $input->setArgument('name', $art_name);
+        $io            = new SymfonyStyle($input, $output);
+        $art_name      = $input->getArgument('name');
+        if ( ! $art_name) {
+            $io->title('Available Art');
+            $io->listing($available_art);
         }
     }
 
     /**
+     * Retrieve the contents of an art file.
+     *
      * @param $name
+     *
      * @return string
      */
     protected function retrieveArt($name)
     {
-        $file_path = $this->config->get('assets_dir') . "/{$name}.txt";
-        if ($this->validateAsset($file_path)) {
-            $output = base64_decode(file_get_contents($file_path));
+        if (file_exists($this->getFilename())) {
+            return base64_decode(file_get_contents($this->getFilename()));
         } else {
-            $output = "Not a valid work of art!";
+            return "<error>There is no source for the requested {$name} artwork.</error>";
```

Comment:
`file_get_contents()` doesn't throw an exception that i can catch unless I set a custom handler. Would it be better to put that together so I can try/catch or would it be better `if(){} else { throw }`?